### PR TITLE
Make Odoacer's revolt landless

### PR DESCRIPTION
**Pull request by IhateTrains**
Please test before merging.

### CODE REVIEW

File: WTWSMS/history/titles/d_odoacer.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+476.1.1={
+	holder = 55 # Odoacer
+	set_global_flag = italy_bureaucratic
+}
```
Comment (loup99) [inline WTWSMS/history/titles/d_odoacer.txt:4]: If Odoacer does not hold Italy at start I don't think he should be Bureaucratic, in my view he would adopt this government after winning. In the pull request he is pre-feudal, he should be Sub-Roman.

File: WTWSMS/common/landed_titles/EBS_titles.txt [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	short_name = yes
+	
+	# Cannot be held as a secondary title
+	primary = yes
+	
+	# Cannot be vassalized
+	independent = yes
+
+	name_tier = KING
+}
```
Comment (loup99) [inline WTWSMS/common/landed_titles/EBS_titles.txt:27]: I'm not sure we need this separate file could be added at the end of the WTWSMS one for similar titles

**Review by loup99 — COMMENTED** [review]
Good work, but a minor issue is that Odoacer remains at war with a landless Romulus for years after taking Italy.

File: WTWSMS/common/landed_titles/EBS_titles.txt (hunk 2 above)
Comment (IhateTrains) [reply WTWSMS/common/landed_titles/EBS_titles.txt]: I added it under k_italy.

File: WTWSMS/history/titles/d_odoacer.txt (hunk 1 above)
Comment (IhateTrains) [reply WTWSMS/history/titles/d_odoacer.txt:4]: > His title is also kept after the victory, it should disappear at that point.

It was supposed to disappear, lemme check.

Comment (loup99) [reply WTWSMS/history/titles/d_odoacer.txt:3]: I looked in history, I didn't try winning

Comment (IhateTrains) [reply WTWSMS/history/titles/d_odoacer.txt:6]: I think I've fixed the aforementioned issues.

**Comment by loup99** [comment]
The issue with Odoacer remaining at war with a landless Romulus for years after taking Italy remains, although the other issues have indeed been fixed.

**Comment by IhateTrains** [comment]
> The issue with Odoacer remaining at war with a landless Romulus for years after taking Italy remains

I couldn't reproduce this. Does this happen every time for you? Are you sure Romulus didn't just declare as a landless claimant?

**Comment by loup99** [comment]
> > The issue with Odoacer remaining at war with a landless Romulus for years after taking Italy remains
> 
> I couldn't reproduce this. Does this happen every time for you? Are you sure Romulus didn't just declare as a landless claimant?

No, it is there in history from start, if I start with Odoacer at a later date.

**Comment by loup99** [comment]
ck2_69

**Comment by IhateTrains** [comment]
Weird, I don't see this war if I start in 477.
image

The war is scripted to end in 476:
```
name = "Odoacer's Invasion of Rome"

casus_belli = {
	casus_belli = odoacer_takeover_war
	actor = 55
	recipient = 96054
	landed_title = e_italy
	date = 476.1.1
}

476.1.1 = {
	add_defender = 96054
	add_attacker = 55
	start_war = yes
}

476.9.4 = {
	rem_attacker = 55
	rem_defender = 96054
	end_war = yes
}
```
Hovewer, the version from French submod has an error:
```
name = "L'invasion d'Odoacre de Rome"

casus_belli = {
	casus_belli = odoacer_takeover_war
	actor = 55
	recipient = 96054
	landed_title = e_italy
	date = 476.1.1
}

476.1.1 = {
	add_defender = 96054
	add_attacker = 55
	start_war = yes
}

479.1.1= {
	rem_attacker = 55
	rem_defender = 96054
	end_war = yes
}
```

**Comment by loup99** [comment]
Ah, then that must be because I activated that sub-mod, sorry for the confusion, didn't realise it would modify this.

**Review by loup99 — APPROVED** [review]

**Pull request merged**